tests/test_model_executor.py

```diff
@@ -126,33 +131,37 @@ def test_imports_and_init_generators(self):
             CONFIG_FILE_KEY_GENERATE_ARGS_INPUT_LATENT_VECTOR_SIZE
         )
 
-    @pytest.mark.parametrize("models_with_args", [models_with_args])
+    # @pytest.mark.parametrize("models_with_args", [models_with_args])
+    @pytest.mark.parametrize("models_with_args", model_ids)
```

Comment:
It's quite important to test model-specific generate function parameters (apart from each model's default). This functionality is now commented out. Was this only a temporal change for quicker testing?

Reply:
Yes, it was a temporal change, mostly because parametrization allows to have clean logs with one test per model. Natively, when one model fails the whole testing of other models in this method is canceled so it's hard to distinguish the issues. Anyway, I will revert those changes shortly.